src/lowertest-derive/src/lib.rs

```diff
@@ -141,7 +122,7 @@ pub fn gen_reflect_info_func(input: TokenStream) -> TokenStream {
                 if let TokenTree::Ident(ident) = tt {
                     let type_name = ident.to_string();
                     let typ = syn::Ident::new(&type_name, Span::call_site());
-                    Some(quote! { enum_dict.insert(#type_name, #typ::mz_enum_reflect()) })
+                    Some(quote! { #typ::add_to_reflect_type_info(&mut result) })
```

Comment:
This is fixed to `add_to_reflected_type_info` in a subsequent commit. Leaving this comment just in case someone else is reading this sequentially and gets confused here.

Reply:
Sorry about that. I've now fixed it in the commit itself.

Reply:
Oh--was really no big deal! Sorry for the unnecessary busywork.